providers/base/units/benchmarks/jobs.pxu

```diff
@@ -8,7 +8,14 @@ template-id: benchmarks/disk/hdparm-read_name
 flags: also-after-suspend
 estimated_duration: 15.0
 user: root
-command: hdparm -t /dev/{name}
+command: 
+ # Check if the disk size is smaller than the threshold (2MB)
+ disk_size=$(blockdev --getsize64 /dev/{name})
+ if [ "$disk_size" -gt 2097152 ]; then
+    hdparm -t /dev/{name}
+ else
+    echo "Disk size is smaller than the threshold (2MB), ignoring the test"
```

Comment:
I don't want to be pedantic, but this doesn't pass the test. It makes it pass. To get the test to be skipped you can either write something smart in the resource expression or write here in 
```
    echo "{ 'outcome' : 'skip', 'comment' : 'Disk size is smaller than the threshold (2MB), ignoring the test'}" > $PLAINBOX_SESSION_SHARE/__result
```

Reply:
I didn't know that that feature existed. It makes more sense indeed to skip these tests. I will do it like that.

Reply:
I agree with you that the skipped tests should be marked as skipped, not passed. I looked into it last friday but it was not as easy as setting the  __result file, since we are only reading that one when we resume the session